packages/dynamite/dynamite/lib/src/models/type_result/object.dart

```diff
@@ -9,10 +9,11 @@ class TypeResultObject extends TypeResult {
     super.generics,
     super.nullable,
     super.isTypeDef,
-  }) : assert(
+  })  : assert(
```

Comment:
It would complain if I remove it.
This has something to do with the super call being added (looks like the formatter tries to align it with the assert).